customize_image.sh

```diff
@@ -1,10 +1,12 @@
 #!/bin/bash
 # customize raspbian image for revolution pi
 
-if [ "$#" != 1 ] ; then
-	echo 1>&1 "Usage: `basename "$0"` <image>"
-	exit 1
-fi
+usage () {
+	echo 'usage: customize_image.sh [-m, --minimize | -h, --help] <image>
+  -m, --minimize	build the minimized image
```

Comment:
Add an explanation what this means. The current help is as useful as: Glass is transparent because it is made out of glass.

Reply:
could you give an explanation?

Reply:
Should those information be kept in Release Node?   It almost the description of difference between "normal" image and "minimal" image, would be relative long and be changed possibly for every Release.